sys/fs/fuse/fuse_vnops.c

```diff
@@ -2665,7 +2667,7 @@ fuse_vnop_getextattr(struct vop_getextattr_args *ap)
 	if (err != 0) {
 		if (err == ENOSYS) {
 			fsess_set_notimpl(mp, FUSE_GETXATTR);
-			err = EOPNOTSUPP;
+			err = EXTERROR(EOPNOTSUPP, "This daemon does not implement extattrs");
```

Comment:
Use "server" for consistency.  And spell out "extended attributes".
```suggestion
		return (EXTERROR(EOPNOTSUPP, "This server does not implement extended attributes"));
```